pycascades-2018/videos/brett-cannon-setting-expectations-for-open-source-participation.json

```diff
@@ -0,0 +1,31 @@
+{
+  "copyright_text": "Standard YouTube License",
+  "description": "Burn-out is common in open source due to a misunderstanding of what users, contributors, and maintainers should expect from each other. This talk will set out to help everyone set reasonable expectations of each other in order to make open source pleasant for everyone involved.",
+  "duration": 1513,
+  "language": "eng",
+  "recorded": "2018-01-22",
+  "related_urls": [
+    {
+      "label": "event schedule",
+      "url": "https://www.pycascades.com/schedule/"
+    },
+    {
+      "label": "slides",
+      "url": "https://onedrive.live.com/redir?resid=ECC96F4CE3E6DAE2!638&authkey=!AGdKnDqu4ZRs-MM&ithint=file%2cpptx"
+    }
+  ],
+  "speakers": [
+    "Brett Cannon"
+  ],
+  "tags": [
+    "community"
+  ],
+  "thumbnail_url": "https://i.ytimg.com/vi/COULfomh-Nk/maxresdefault.jpg",
+  "title": "Setting Expectations for Open Source Participation",
+  "videos": [
+    {
+      "type": "youtube",
+      "url": "https://www.youtube.com/watch?v=COULfomh-Nk"
```

Comment:
https://www.youtube.com/watch?v=-Nk-8fSJM6I

Reply:
The URL in [this lines](https://github.com/Daniel-at-github/data/blob/7e892b8d8160ed5f1663d9e1548db33429be489b/pycascades-2018/videos/dhruv-govil-python-for-feature-film.json#L22-28)
~~~ json
  "title": "Python for Feature Film",
  "videos": [
    {
      "type": "youtube",
      "url": "https://www.youtube.com/watch?v=s6i2FG0Y1oA"
    }
  ]
~~~
Is now https://www.youtube.com/watch?v=qGhKvEA2aNo

Reply:
... and some more.
Hope it's OK now.